src/grid.js

```diff
@@ -321,3 +321,44 @@ export const getParentTile = (level, tile) => {
 		return null;
 	}
 };
+
+/**
+ * Return child tiles for given level and tile
+ * @param level {number}
+ * @param tile {Array}
+ * @return {Array}
+ * TODO @vlavh1989 tests
+ */
+export const getChildTiles = (level, tile) => {
+	if (level < 23 && tile) {
```

Comment:
Number of nevels is defined in constants/grid.js.

Reply:
Fixed: https://github.com/gisat-panther/ptr-tile-grid/pull/25/commits/2363102b636964a25b5bfca7a91f655077bc26ed